com.unity.render-pipelines.high-definition/Editor/Material/MaterialHeaderScope.cs

```diff
@@ -26,6 +26,15 @@ internal struct MaterialHeaderScope : IDisposable
         int oldIndentLevel;
 #endif
 
+        /// <summary>
+        /// Creates a material header scope to display the foldout in the material UI.
+        /// </summary>
+        /// <param name="title">Title of the header.</param>
+        /// <param name="bitExpanded">Bit index to use for the.</param>
+        /// <param name="materialEditor">The current material editor.</param>
+        /// <param name="spaceAtEnd">Do the block shows a space at the bottom of it's UI.</param>
+        /// <param name="colorDot">Display a colored dot like in the layered UI material</param>
+        /// <param name="subHeader">Is sub-header, affects the style of the header</param>
```

Comment:
Set this to true to make this block into a sub-header. This affects the style of the header. Set to false to make this block use the standard style.